alpaca_trade_api/stream.py

```diff
@@ -136,7 +148,7 @@ def _subscribe(self, handler, symbols, handlers):
         if self._running:
             asyncio.get_event_loop().run_until_complete(self._subscribe_all())
 
-    async def _unsubscribe(self, trades=(), quotes=(), bars=()):
+    async def _unsubscribe(self, trades=(), quotes=(), bars=(), daily_bars=()):
```

Comment:
Please add `daily_bars` to the message that is sent

Reply:
Great catch, thank you!